python/flexflow_c.cc

```diff
@@ -1734,7 +1734,7 @@ DLRMConfig::DLRMConfig(void)
       continue;
     }
     if (!strcmp(argv[i], "--arch-embedding-size")) {
-      std::stringstream ss(std::string(argv[++i]));
+      std::stringstream ss((std::string(argv[++i])));
```

Comment:
why do we need another `()`?

Reply:
Without the additional (), it gets parsed as a function prototype

Reply:
https://github.com/flexflow/FlexFlow/pull/392#discussion_r1013553783